.github/workflows/tox.yaml

```diff
@@ -26,10 +26,10 @@ jobs:
             toxenv: py310-core
           - python-version: "3.10"
             toxenv: py310-integration
-          - python-version: "3.11"
-            toxenv: py311-core
-          - python-version: "3.11"
-            toxenv: py311-integration
+          # - python-version: "3.11"
+          #   toxenv: py311-core
+          # - python-version: "3.11"
+          #   toxenv: py311-integration
```

Comment:
Should we maybe just remove it instead of commenting it out?

Reply:
Not sure; do we have plans to support it? I guess the cleanest thing is to actually solve whatever is causing 3.11 to not work, but on the other hand maybe it doesn't matter

Reply:
I really would remove it before merging - having this just commented out is not a good style.